scripts/ci/benchmarks-ci.sh

```diff
@@ -21,6 +21,8 @@ pallets=(
 	frame_system
 )
 
+mkdir -p $benhcmarkOutput
```

Comment:
I suggest we fix the typo as another PR fixes it and that will introduce conflicts.

```suggestion
mkdir -p $benchmarkOutput
```

Reply:
See #1300